core/services/ocrcommon/transmitter.go

```diff
@@ -62,11 +77,11 @@ func (t *transmitter) FromAddressForTransaction() common.Address {
 	// Use Round-Robin to select the next fromAddress.
 	nextFromAddress := t.fromAddresses[t.nextFromAddressIndex]
 
-	// Only apply round-robin logic for multiple sending keys.
```

Comment:
I think we should just always use GetRoundRobinAddress(), it'll always do the right thing even if num keys is 1.

Reply:
And if it doesn't, the method is buggy and should be fixed.